mitmproxy/addons/dumper.py

```diff
@@ -165,8 +170,8 @@ def _echo_request_line(self, flow: http.HTTPFlow) -> None:
 
         http_version = ""
         if (
-                flow.request.http_version not in ("HTTP/1.1", "HTTP/1.0")
-                or flow.request.http_version != getattr(flow.response, "http_version", "HTTP/1.1")
+            flow.request.http_version not in ("HTTP/1.1", "HTTP/1.0")
+            or flow.request.http_version != getattr(flow.response, "http_version", "HTTP/1.1")
```

Comment:
Please use the new fairly new helper functions: `flow.request.is_http11` and similar.
(also below again)